test/unit/api_spec.js

```diff
@@ -1760,6 +1760,30 @@ sources, for full support with Dvips.`)
       await loadingTask.destroy();
     });
 
+    it("gets text content, with negative spaces (bug 931481)", async function () {
```

Comment:
To fix the GitHub-actions failure, please add the following block at the top of this test:
```javascript
if (isNodeJS) {
  pending("Linked test-cases are not supported in Node.js.");
}
```

Reply:
Thank you, I was exactly looking for that.